tidb-configuration-file.md

```diff
@@ -362,6 +362,8 @@ Configuration items related to performance.
 - Default value: `6291456` (in bytes)
 - The size limit of a single key-value record in a transaction. If the size limit is exceeded, TiDB returns the `entry too large` error. The maximum value of this configuration item does not exceed `125829120` (120 MB).
 - Note that TiKV has a similar limit. If the data size of a single write request exceeds [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size), which is 8 MB by default, TiKV refuses to process this request. When a table has a row of large size, you need to modify both configurations at the same time.
+- The default `max_allowed_packet` is 67108864 (64 MiB), which may lead to truncation with rows that are bigger.
+- The default `txn-total-size-limit` is 100 MiB, which you need to increase if you increase `txn-entry-size-limit` to be over 100 MiB.
```

Comment:
What does "rows that are bigger" mean? Does it mean rows bigger than `txn-entry-size-limit`?
```suggestion
- Because the default value of [`max_allowed_packet`](/system-variables.md#max_allowed_packet) (the maximum size of a packet for the MySQL protocol) is 67108864 (64 MiB), which might lead to truncation with rows that are bigger.
- The default value of [`txn-total-size-limit`](#txn-total-size-limit) (the size limit of a single transaction in TiDB) is 100 MiB. If you increase the `txn-entry-size-limit` value to be over 100 MiB, you need to increase the `txn-total-size-limit` value accordingly.
```

Reply:
If a column is larger than `max_allowed_packet` it gets truncated. With a strict SQL mode it will probably give an error instead of truncating the value, but I didn't test that.